{"uuid": "93c8c634-6b29-4916-ad8d-b056f11199b8", "children": ["1534cc1e-e5e3-4518-85c9-89809665895e", "69e20363-6eee-46b5-8f99-fb4d22f654f8"], "befores": [{"name": "supervisor_chain", "status": "broken", "statusDetails": {"message": "KeyError: 'llama-3.2'\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/OA/OA/tests/supervisor_chain.py\", line 19, in supervisor_chain\n    llm = get_available_providers()[model]\n          ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^\n"}, "start": 1730426848927, "stop": 1730426849513}], "start": 1730426848927, "stop": 1730426850157}